sources/packages/web/src/components/aest/institution/modals/AssessOfferingModal.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <ModalDialogBase
+    :showDialog="showDialog"
+    @dialogClosed="dialogClosed"
+    :title="title"
+  >
+    <template v-slot:content>
+      <div class="mt-2">
+        <formio
+          formName="approvedeclineoffering"
+          :data="{ offeringStatus: offeringStatus }"
```

Comment:
Creating a new object like this will not stop the reactiveness?
Is it not better to create a computed?

Reply:
It is not required to be reactive here because, this value once assigned to model doesn't change.